SPM Example/PortalSwift/ViewController.swift

```diff
@@ -1062,12 +1062,12 @@ class ViewController: UIViewController, UITextFieldDelegate {
             self.stopRefreshBalanceTimer()
           }
 
-          let availableRecoveryMethods = try await self.portal?.availableRecoveryMethods() ?? []
-          let walletExists = try await self.portal?.doesWalletExist() ?? false
+          let availableRecoveryMethods = try await self.portal?.availableRecoveryMethods(nil) ?? []
```

Comment:
What changed that is causing us to force nil being passed here? Can we make it optional so it's not a breaking change